src/routes/index.js

```diff
@@ -0,0 +1,21 @@
+const { Router } = require("express");
+// Importar todos los routers;
+// Ejemplo: const authRouter = require('./auth.js');
+
+const router = Router();
+
+// Configurar los routers
+// Ejemplo: router.use('/auth', authRouter);
+
+
+router.get("/test", async (req, res) => {
+  res.status(200).send({ hi: "Hello world!!!" })
+});
+
+router.all("*", (req, res) => {
+  res.redirect("/");
+});
+
+router;
```

Comment:
que hace este router; solo?

Reply:
si estas desde el cliente y escribís en la barra de direcciones cualquier ruta te lleva a la Raiz devolviéndote el FrontEnd. Sirve para que cuando pasan una ruta sin querer desde el lado cliente que pertenece a React-Router Express no te devuelva un "Can not get <ruta>" sino que te tire de nuevo al FrontEnd